CONTRIBUTING.md

```diff
@@ -14,10 +14,16 @@ If you want to contribute code fixes, see below.
 ### Install dependencies
 
 ```sh
-$ git clone https://github.com/TemporalAgent7/datacore
+$ git clone https://github.com/stt-datacore/website.git
 $ npm install 
 ```
 
+In order to get images on your server you have to run the command. 
+```sh
+$ cp .env.default .env
+```
+Then change the line "GATSBY_ASSETS_URL=" to "GATSBY_ASSETS_URL=http://assets.datacore.app/".
```

Comment:
This should probably suggest changing it to point to wherever the person is hosting asset-server, or if they're pointing to us, it should be an https link

Reply:
My thinking is that the vast majority of potential contributors won't have any interest in the asset server. Its a work horse where nothing sexy happens. Plus this document is about how to contribute to the website. Ideally there should be a parallel document on that repository explaining how to setup a local version. Then that can linked here in a second sentence. Until that time maybe adding a sentence like "If you are using a local version of the asset server then use "GASTBY_ASSETS_URL=local_dir" could work.